cmake/FPrime.cmake

```diff
@@ -62,6 +62,7 @@ function(fprime_setup_global_includes)
     include_directories("${FPRIME_FRAMEWORK_PATH}")
     include_directories("${FPRIME_CONFIG_DIR}")
     include_directories("${FPRIME_PROJECT_ROOT}")
+    include_directories("${FPRIME_PROJECT_ROOT}/lib/include")
```

Comment:
This is not required, users can always 
```cmake
target_include_directories(${FPRIME_CURRENT_MODULE} PUBLIC "/path/to/lib/include")
```
but it felt like it could be a QOL improvement. Happy to take it out if we feel uneasy about having this at the global level.

Reply:
This would only be useful in the case where a user is being provided with pre-built `lib.a` library along with some `include/<libName>/[...]` files (unsure if that's common?). If the library is being built from source, installed includes should go in the build cache.